src/main/java/com/flightstats/hub/spoke/ReadOnlyRemoteSpokeStore.java

```diff
@@ -0,0 +1,78 @@
+package com.flightstats.hub.spoke;
+
+import com.flightstats.hub.dao.QueryResult;
+import com.flightstats.hub.metrics.Traces;
+import com.flightstats.hub.model.Content;
+import com.flightstats.hub.model.ContentKey;
+import com.google.inject.Inject;
+
+import java.net.UnknownHostException;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.SortedSet;
+
+public class ReadOnlyRemoteSpokeStore implements RemoteSpokeStore {
+    private final RemoteSpokeStore delegate;
+
+    @Inject
+    public ReadOnlyRemoteSpokeStore(RemoteSpokeStore delegate) {
+        this.delegate = delegate;
+    }
+
+    @Override
+    public boolean insert(SpokeStore spokeStore, String path, byte[] payload, String spokeApi, String channel) {
```

Comment:
Yeah...I'd love to refactor this back to a single cache with normal read/write-through behavior...but that seems like a task for another branch.
Since we're breaking the abstraction anyway with that SpokeStore param, maybe it would be more clear to create explicit insertToWriteStore/insertToReadStore methods?

Reply:
I'd be in favor of that. 👍

Reply:
Well...that snowballed a bit, but it's more obvious how we're using spoke now and the r/o overrides are only applied to write spoke operations.